test/transforms.jl

```diff
@@ -654,6 +654,50 @@
     @test n1 == n2
   end
 
+  @testset "Filter" begin
+    # Test with _check_no_missing_in_row
+    col1 = [1, 2, 3, 4, 5]
+    col2 = [missing, 2, 3, 4, 5]
+    col3 = [5, 5, 5, 5, 5]
+    col4 = [1, 2, 3, 4, 5]
+    col5 = [1, 2, missing, 4, 5]
+    table = Table(;col1, col2, col3, col4, col5)
+
+    function _check_no_missing_in_row(row)
+      for el in row
+          el === missing && return false
+      end
+  
+      return true
+    end
```

Comment:
What is this function doing here in the tests?

Reply:
I see that you are using it once, but you shouldn't call it _check_no_missing_in_row because this is a convention used for internal functions. Try to be more pedantic with code style.